frontend/src/views/ClientView.vue

```diff
@@ -6,47 +6,78 @@ import Dropdown from "primevue/dropdown";
 import FloatLabel from "primevue/floatlabel";
 import InputText from "primevue/inputtext";
 import Textarea from "primevue/textarea";
+import { useConfirm } from "primevue/useconfirm";
+import { useToast } from "primevue/usetoast";
 import { onMounted, ref } from "vue";
+import { computed } from "vue";
 import { useRoute, useRouter } from "vue-router";
 
 import { createClient, deleteClient, getClient, updateClient } from "@/backendClient";
 import { ClientSalutation } from "@/global/types/appTypes";
 import type { ClientCreate, ClientUpdate } from "@/global/types/dataEditTypes";
 import { ROUTES } from "@/router";
 
+const confirm = useConfirm();
+const toast = useToast();
+const onDeleteClient = async () => {
+  await deleteClient(`${route.params.id}`);
+  router.push(`${ROUTES.CLIENT.path}`);
+};
+const confirmDelete = () => {
+  confirm.require({
+    message: "Are you sure you want to delete this client?",
+    header: "Confirmation",
+    rejectLabel: "Cancel",
+    acceptLabel: "Delete",
+    accept: async () => {
+      await onDeleteClient();
+      toast.add({
+        severity: "info",
+        summary: "Confirmed",
+        detail: "You have deleted this client",
+        life: 3000,
```

Comment:
See here: https://github.com/fr-ser/scaffolding-management/pull/16/files#r1684544661

Maybe we can extract this method if we reuse it in two places? 🤔

Reply:
We have this code in two places, right? Can we extract it?